var/spack/repos/builtin/packages/libnrm/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libnrm(AutotoolsPackage):
+    """Libnrm, the application instrumentation library for the Node
+    Resource Manager(NRM)."""
+
+    homepage = "https://xgitlab.cels.anl.gov/argo/libnrm"
+    url = "https://www.mcs.anl.gov/research/projects/argo/downloads/libnrm-0.1.1.tar.gz"
+    version('0.1.0', '2edae952edce3df9f1562ea64d4ae201')
```

Comment:
You have a URL for version 0.1.1 but a version for 0.1.0. Can you make them match? It won't affect anything if they don't, but `spack url` has some testing commands to see how good we are at version parsing and these will be affected.

Reply:
good catch, thank you.